packages/terra-toggle/src/terra-dev-site/test/toggle/DefaultToggle.test.jsx

```diff
@@ -14,9 +14,10 @@ class ToggleDefault extends React.Component {
   }
 
   render() {
+    const { isOpen } = this.state;
     return (
       <div>
-        <button type="button" id="trigger-toggle" onClick={this.handleOnClick}>Default Toggle</button>
+        <button type="button" id="trigger-toggle" onClick={this.handleOnClick} aria-expanded={isOpen}>Default Toggle</button>
```

Comment:
Let's add the aria-controls and id similar to `packages/terra-toggle/src/terra-dev-site/doc/example/AnimatedToggle.jsx`

Reply:
Resolved in 3478f79